src/BornOnThisDay.java

```diff
@@ -0,0 +1,145 @@
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+
+public class BornOnThisDay {
+    public static void main(String[] args) throws IOException, JSONException {
+        System.out.println(convertAndFilterResponse(getResponse("04", "08")));
+    }
+    /**
+     * BornOnThisDay.getResponse()
+     * Sends HTTP request for 'births' to Wikimedia's 'On this day' api. The response is a JSON object,
+     * with a value of a JSON Array representing people born on the given day.
+     * The response is read in and converted to a String
+     *
+     * @param mm a two-character String representing the two digit month given by the user to be appended
+     *           to the url path.
+     *
+     * @param dd a two character String representing the two digit day given by the user to be appended
+     *           to the url path
+     *
+     * @return the response converted to a String
+     *
+     * @author Bradley Boggs, bradleywboggs@gmail.com
+     */
+
+    public static String getResponse(String mm, String dd) {
+
+        try {
+            String url = "https://en.wikipedia.org/api/rest_v1/feed/onthisday/births/" + mm + "/" + dd;
+            URL obj = new URL(url);
+            HttpsURLConnection con = (HttpsURLConnection) obj.openConnection();
```

Comment:
Consider moving these outside of the try/catch block unless these lines can actually throw an exception. Try/catch blocks should usually only contain the code that can throw an exception.